src/lib.rs

```diff
@@ -730,17 +730,34 @@ mod tests {
     #[macro_export]
     macro_rules! assert_errs {
         ($result:expr, [ $( $kind:pat ),* ]) => {
-            eprintln!("{} => {:?}", stringify!($result), $result);
-            assert!(matches!(
-                $result.unwrap_err().as_slice(),
+            let errs = if let Err(errs) = $result {
+                errs
+            } else {
+                eprintln!("    PROGRAM SUCCEEDED, WHEN IT SHOULD FAIL!");
+                unreachable!();
```

Comment:
I don't think the tests should shout, even if they fail.
```suggestion
                eprintln!("    Program succeeded when it should've failed");
                unreachable!();
```